discovery-infra/download_logs.py

```diff
@@ -80,9 +81,11 @@ def get_clusters(client, all_cluster):
     return client.clusters_list()
 
 
-def should_download_logs(cluster: dict):
-    return cluster['status'] in [ClusterStatus.ERROR]
```

Comment:
We should probably add a `add_triage_signature.py` signature to detect non-error state cluster and give them a special label so they can be easily filtered in triage dashboards

Reply:
@osherdp